### DISCUSSION TITLE
part-ii/2021-18/

### DISCUSSION THREAD

**Discussion by tripos-comments** [tripos-education/maths-tripos-questions#412] [Question comments]
Part II 2021 Applied Probability: Paper 3, Section II, $27 \mathrm{~K}$ 

https://questions.tripos.org/part-ii/2021-18/

**Comment by ArthurConmy** [comment]
<details>
<summary>a)</summary>

I get that if $r<1$ there is an invariant distribution, and if $r=1$ then if $\lambda < 2 \mu$ there's an invariant distribution.
</details> 

<details>
<summary>b)</summary>

I get that there's always an invariant distribution and that it is proportional to $\pi_i = 1/\mu^i$
</details> 

<details>
<summary>c)</summary>


It's a birth-and-death chain so satisfies detailed balance so the chain is reversible in equilibrium. Repeat Burke's theorem proof.
</details>
